pkg/tidb/forwarder.go

```diff
@@ -93,10 +92,8 @@ func (f *Forwarder) pollingForTiDB() {
 			return err
 		}, bo)
 		if err != nil {
-			if errorx.IsOfType(err, ErrNoAliveTiDB) {
```

Comment:
This may mean that there is connection error to the PD. In this case TiDB can be still alive and I think there is no need to clear TiDB remotes.